CMakeLists.txt

```diff
@@ -88,9 +88,14 @@ if (FL_BUILD_LIBRARIES)
     include(${FL_BINDING_PYTHON}/CMakeLists.txt)
   endif ()
 
+  if (NOT FL_LIBRARIES_USE_MKL)
+    find_package(OpenMP) # Needed for linking with OpenMP_CXX when MKL OFF
```

Comment:
reminder: MKL is usually coming with iomp (the Intel OpenMP impl), meaning if not using MKL, then gomp (the Gnu OpenMP impl, part of most/all gcc) seems to be needed.

Reply:
This is a reasonable thing to do. Right now we [already link OpenMP](https://github.com/facebookresearch/flashlight/blob/5b0771abdbcf1f39037b3a7d322faebfd4eca43b/flashlight/lib/audio/feature/CMakeLists.txt#L25) when building flashlight libraries, so I'd like to have one source of truth for doing this, especially since we might mix iomp and non-Intel OpenMP, which isn't good.

This is pretty unrelated to the RNN classification example. @WilliamTambellini can you send separate PRs for each of these fixes so they can be reviewed separately? That'll make it much easier to reason about everything.

Reply:
Now a separate PR:
https://github.com/facebookresearch/flashlight/pull/285
Remember that master does not seem to be able to  link when MKL off.